packages/frontend/src/app/submissions/page.tsx

```diff
@@ -0,0 +1,177 @@
+                  <td className="p-0 table-fixed break-all">
+                    <div className="flex p-4 bg-white mb-1 items-center rounded-l-lg">
+                      <Image
+                        className="mr-2"
+                        src={item.logo_url}
+                        alt={item.name}
+                        width={24}
+                        height={24}
+                      />
+                      <p className="leading-6 min-h-6 h-6 overflow-hidden">
+                        {item.name}
+                      </p>
+                    </div>
+                  </td>
+                  <td className="p-0 table-fixed break-all">
+                    <div className="p-4 bg-white mb-1">
+                      <p className="leading-6 min-h-6 h-6 overflow-hidden text-right capitalize">
+                        {item.state}
+                      </p>
+                    </div>
+                  </td>
+                  <td className="p-0 table-fixed break-all">
+                    <div className="p-4 bg-white mb-1 ">
+                      <p className="leading-6 min-h-6 h-6 overflow-hidden text-right">
+                        {dayjs(item.updatedAt).format('DD/MM/YYYY')}
+                      </p>
+                    </div>
+                  </td>
+                  <td className="p-0 table-fixed break-all">
+                    <div className="p-4 bg-white mb-1">
+                      <p className="leading-6 min-h-6 h-6 overflow-hidden text-right">
+                        {dayjs(item.createdAt).format('DD/MM/YYYY')}
+                      </p>
+                    </div>
+                  </td>
+                  <td className="p-0 table-fixed break-all">
+                    <div className="p-4 bg-white mb-1 rounded-r-lg">
+                      <p className="leading-6 min-h-6 h-6 overflow-hidden text-right">
+                        {item.email}
+                      </p>
+                    </div>
+                  </td>
+                </tr>
+              )
+            })}
+        </TableBody>
+      </Table>
+    </PageContainer>
+  )
+}
+
+interface FilterProps {
+  options: string[]
+  selectedOptions: string[]
+  onChange: (value: string[]) => void
+}
+
+interface FilterButtonProps {
+  onClick: any
+  isActive: boolean
+  value: string
+}
+
+function Filter({ options, selectedOptions, onChange }: FilterProps) {
+  function toggleFilterValue(option: string): void {
+    if (selectedOptions.includes(option)) {
+      const optionIndex = selectedOptions.indexOf(option)
+
+      selectedOptions.splice(optionIndex, 1)
+      onChange([...selectedOptions])
+    } else {
+      onChange([...selectedOptions, option])
+    }
+  }
+
+  function isActive(filter: string): boolean {
+    return selectedOptions.includes(filter)
+  }
+
+  return (
+    <div className="flex flex-col">
+      <div className="pl-4 h-6 mb-2">
+        <TextSmBold className="h-6">Filters</TextSmBold>
```

Comment:
Could we find out a way that we can style it with parent and not set `h-6` to the text element?